addons/test_mail_full/tests/test_mass_mailing_statistics.py

```diff
@@ -63,7 +63,7 @@ def test_mailing_statistics_sms(self):
         kpi_values = body_html.xpath('//div[@data-field="sms"]//*[hasclass("kpi_value")]/text()')
         self.assertEqual(
             [t.strip().strip('%') for t in kpi_values],
-            ['100', str(mailing.opened_ratio), str(mailing.replied_ratio)]
+            ['100', str(mailing.opened_ratio), str(mailing.bounced_ratio)]
```

Comment:
Isn't the second one "click_ratio" , looking at "_prepare_statistics_email_values" ?

Reply:
@tde-banana-odoo yes, i updated the code

Reply:
But you did not update commit message ^^

Reply:
ahah no, sorry forgot 1 sec